src/robust.jl

```diff
@@ -21,7 +21,7 @@ function uplo(x::AbstractVector; prop::Real=0.0, count::Integer=0)
     end
 
     # indices for lowest count values
-    x2 = copy(x)
+    x2 = collect(x)
```

Comment:
Perhaps we should use `copymutable` here?

Reply:
I have also changed the call to `partialsort!` as just requesting `count+1` and `n-count` positions should be enough and it is a bit faster.